DependencyInjection/FOSCommentExtension.php

```diff
@@ -54,6 +56,29 @@ public function load(array $configs, ContainerBuilder $container)
             $this->loadAcl($container, $config);
         }
 
+        // To be removed when dependency on Symfony DependencyInjection is bumped to 2.6
+        if (!interface_exists('Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface')) {
+            $container->setDefinition('security.token_storage', new Reference('security.context'));
+        }
+
+        // To be removed when dependency on Symfony DependencyInjection is bumped to 2.6
+        if (!interface_exists('Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface')) {
+            $container->setDefinition('security.authorization_checker', new Reference('security.context'));
```

Comment:
please don't do this:

- setting a reference as Definition is invalid and won't work (I think what you are trying to use is an alias)
- It may break all other bundles trying to detect whether the old API or the new API should be used.

You should change the argument injected in your services instead (a solution to avoid having to alter lots of services would be to define a private alias ``fos_comment.authorization_checker`` and redefining it depending on the availability of the new API)